components/eam/tools/topo_tool/bin_to_cube/bin_to_cube.F90

```diff
@@ -72,11 +70,31 @@ program convterr
   real(r8) :: vol,dx_rad,vol_cube,area_latlon,darea_latlon       ! latitude array
   real(r8), allocatable, dimension(:,:) :: darea_cube
 
+  INTEGER :: iargc
+
+  character(len=1024) :: raw_latlon_data_file,output_file,string_ncube
+  
+  iargc = command_argument_count()
+  if (iargc /= 3) then
+      print *, "Usage: <executable> raw_latlon_data_file output_file ncube"
+      stop
+  end if
+
+  call get_command_argument(1, raw_latlon_data_file)
+  call get_command_argument(2, output_file)
+  call get_command_argument(3, string_ncube)
+  read(string_ncube, *) ncube
```

Comment:
I'd make this a command argument too if you're already making the others command arguments. Interactive scripts make it harder to pipe all this into larger script workflows.